test/acceptance/features/steps/steps.py

```diff
@@ -278,6 +278,40 @@ def then_envFrom_contains_intermediate_secret_name(context, intermediate_secret_
         intermediate_secret_name, wait=True, timeout=120) is not None, f"There is no deployment with intermediate secret {intermediate_secret_name}"
 
 
+# STEP
+@given(u'OLM Operator Backend is running')
+def backend_operator_manifest_installed(context):
+    openshift = Openshift()
+    _ = openshift.oc_apply_yaml_file(os.path.join(os.getcwd(), "test/acceptance/resources/backend.operator.manifest.yaml"))
+
+
+# STEP
+@given(u'Backend CR "{cr_name}" is applied')
+def create_backend_cr(context, cr_name):
+    create_cr(context, cr_name)
+
+
+# STEP
+@given(u'Service Binding request is applied')
+def create_sbr(context):
+    sbr_is_applied(context)
+
+
+# STEP
+@when(u'Backend status in "{backend_name}" is updated')
+def when_backend_status_updated(context, backend_name):
+    create_cr(context, backend_name)
+
+
+# STEP
+@then(u'"{secret_key}" key in secret "{secret_name}" has "{secret_value}"')
+def then_check_secret_key_value(context, secret_name, secret_key, secret_value):
```

Comment:
remove `then`

Reply:
I referred multiple then steps in the master and they all seem to have this refix https://github.com/redhat-developer/service-binding-operator/blob/master/test/acceptance/features/steps/steps.py#L166 https://github.com/redhat-developer/service-binding-operator/blob/master/test/acceptance/features/steps/steps.py#L181 https://github.com/redhat-developer/service-binding-operator/blob/master/test/acceptance/features/steps/steps.py#L189 https://github.com/redhat-developer/service-binding-operator/blob/master/test/acceptance/features/steps/steps.py#L199

Reply:
good catch - we should fix those things in master as well, but not in this PR. Let's not introduce more cases like this in this PR.